src/collections/blog/2023/2023-05-22-changing-release-channels-in-meshery/index.mdx

```diff
@@ -0,0 +1,177 @@
+  unexpected changes from being introduced into their system. However, doing so cancels out any future subscription-based 
+  benefits such as receiving security patches or bug fixes that were added after that version was released.
+
+  Therefore, it's important for users to weigh the pros and cons of each option before making decisions on how they 
+  want to manage their Meshery deployment. It's recommended that organizations have a well-defined upgrade 
+  strategy based on testing and validation procedures prior to applying new releases in production environments whether 
+  via subscriptions or manual upgrades to ensure 
+  that system availability is maintained and risks are minimized.
+</p>
+<p>
+  There are two ways to switch between Meshery release channels: using
+  mesheryctl or by editing your meshconfig file. In this blog post, we'll cover
+  both methods.
+</p>
+
+<br />
+<h3>
+  What is Meshconfig?
+</h3>
+<p>
+  Meshconfig is a configuration file that is used to configure Meshery. It is
+  typically located in the <code>~/.meshery</code> directory.It contains
+  information about the current release channel, the version of Meshery that is
+  installed, and other configuration options that are specific to your Meshery
+  installation. It is a YAML file, so you can edit it using any text editor.
+
+  Meshconfig is automatically generated when you run Meshery for the first time.
+  It is also automatically updated when you update Meshery. Meshconfig is a YAML
+  file, so you can edit it using any text editor. 
+</p>
+<h3>
+  Switching between Meshery release channels by editing your meshconfig file.
+</h3>
+
+<br />
+Another way to switch between Meshery release channels is by editing your config.yaml file. Here's how:
+
+<ol>
+<li>
+  Open your config.yaml file in a text editor.
+</li>
+<li>
+  Find the channel field in the meshery section of the file.
+</li>
+<li>
+  Change the value of the version field to the desired release channel, stable
+  or edge.
+</li>
+<li>
+  Save the changes to the file.
+</li>
+
+<li>
+  To confirm that the channel has been changed, run the following command again:
+
+<pre>
+  <code class="language-bash">mesheryctl system channel view</code>
+</pre>
+</li>
+</ol>
+
+<p>
+  After making these changes, future updates to Meshery will use the selected
+  release channel.
+</p>
+
+<br />
+<h3>Switching between Meshery release channels using mesheryctl.</h3>
+<br />
+
+<p>
+  mesheryctl is a command-line tool for managing Meshery. You can use it to switch between different release channels. Here's how:</p>
+<p>
+  Open your terminal and confirm that you have mesheryctl installed by running  <code>mesheryctl version</code>. If you don't have mesheryctl installed, you can install it by following the instructions in the  <a href="https://docs.meshery.io/installation/mesheryctl">Meshery documentation</a>.
+</p>
+
+<ul>
+<li>
+  Run the following command to see the current configuration for
+  Meshery:
```

Comment:
Please remove the errant line breaks throughout this markdown file, allowing line wrap to naturally occur.